R/walk_through_page.R

```diff
@@ -0,0 +1,524 @@
+                withSpinner(
+                  type = 6,
+                  plotOutput(outputId = ns("page6LanganRan"))
+                )
+              )
+            )
+          )
+        ),
+        
+        div(
+          class = 'page',
+          id = ns('page7'),
+          h2("Bayesian versus Frequentist framework"),
+          h4("Up until now, this walk-through was using a frequentist framework. But would there be any difference in using a Bayesian framework?"),
+          br(),
+          h4("Often, researchers like to use a Bayesian framework as it is more flexible towards more complex models."),
+          h4("Below you can see the forest plots for our example random-effects models under a frequentist and Bayesian framework - the estimates are very similar."),
+          h4("With regards to meta-analysis, a main difference between these frameworks is how the between-study heterogeneity is modelled. Under a frequentist framework, it is modelled with a single number. However, under a Bayesian framework, it can be modelled with a distribution to represent the uncertainty in the estimation of the heterogeneity (which can be considerable)."),
+          h4("This will have a knock-on effect on the sample size calculations - an effect that could be argued to be more accurate."),
+          br(),
+          h5("Unfortunately, MetaImpact doesn't currently have Bayesian functionality for the Calculator element, however, this is in development."),
+          br(),
+          fluidRow(
+            align = 'center',
+            column(
+              width = 6,
+              h4("Frequentist"),
+              withSpinner(
+                type = 6,
+                plotOutput(outputId = ns('page7ForestFreq'))
+              )
+            ),
+            column(
+              width = 6,
+              h4("Bayesian"),
+              withSpinner(
+                type = 6,
+                plotOutput(outputId = ns('page7ForestBayes'))
+              )
+            )
+          )
+        ),
+        
+        div(
+          class = 'page',
+          id = ns('page8'),
+          h2("End of Walk-Through"),
+          br(),
+          h4("Thank you for going through this walk-through of the methods underpinning MetaImpact."),
+          h4("You are now welcome to explore the rest of the application, including uploading your own data (or continue using example data that is supplied and usable throughout the app), and using further options within the calculator (such as changing the significance threshold and type of impact desired).")
+        )
+        
+      ),
+      br(),
+      fluidRow(
+        column(
+          width = 1,
+          offset = 4,
+          actionButton(inputId = ns("prevBtn"), label = "< Previous")
+        ),
+        column(
+          width = 1,
+          offset = 2,
+          actionButton(inputId = ns("nextBtn"), label = "Next >")
+        )
+      )
+    )
+  )
+}
```

Comment:
This is still quite big, could it be broken into each walk-through page?

Reply:
I've broken it down into the pages that had server related content (e.g. not the first and last page)